usr/sbin/p11sak/p11sak.c

```diff
@@ -3186,28 +3188,17 @@ int main(int argc, char *argv[])
     /* now try to load the pkcs11 lib (will exit(99) on failure) */
     load_pkcs11lib();
 
-    /* Prompt for PIN if not already set via option */
+    /* no pin in argv, fallback 1: try env */
+    if (!pin)
+        pin = getenv("PKCS11_USER_PIN");
```

Comment:
RFC: rename PKCS11_USER_PIN to OCK_USER_PIN in preparation for the consolidation.

Reply:
hm, well, the external manual documents PKCS11_USER_PIN for p11sak, so changing this is breaking a documented API.....
https://www.ibm.com/docs/en/linux-on-systems?topic=features-environment-variables

Reply:
I was not aware, that the external documentation differs from the man pages of the tools. In this case, we should not do a renaming.

Reply:
Wat about the env variable name for the so pin? PKCS11_SO_PIN or OCK_SO_PIN?

Reply:
My proposal: OCK_USER_PIN and OCK_SO_PIN is supported, as well as PKCS11_USER_PIN. The OCK_USER_PIN variable is preferred over PKCS11_USER_PIN. The man pages document all varaibles, but PKCS11_USER_PIN is marked as deprecated.

And we ask ID for a change in the external documentation.

Reply:
Well, the p11sak man page did not mention the env var at all, so in that terms it was not wrong :-) 

In general I am OK with your approach, although it makes it more complicated at the end....

The other thing is that p11sak (in contrast to the other OCK tools) is supposed to be usable with other PKCS11 implementations as well (you can set env var PKCSLIB to point to a foreign PKCS11 implementation - this env var is also not mentioned in the man page....grrr) . It should in general not be OCK specific. So the name PKCS11_USER_PIN is also more general than OCK_USER_PIN.

So I am not sure of I would not simply leave it as is and keep it as PKCS11_USER_PIN.